src/descriptor/policy.rs

```diff
@@ -1400,45 +1459,138 @@ mod test {
 
         let signers_container = Arc::new(SignersContainer::from(keymap));
 
-        let original_policy = wallet_desc
-            .extract_policy(&signers_container, &secp)
+        let psbt: PSBT = deserialize(&base64::decode(ALICE_SIGNED_PSBT).unwrap()).unwrap();
+
+        let policy_alice_psbt = wallet_desc
+            .extract_policy(&signers_container, BuildSatisfaction::Psbt(&psbt), &secp)
             .unwrap()
             .unwrap();
+        println!("{}", serde_json::to_string(&policy_alice_psbt).unwrap());
```

Comment:
nit, should comment out or remove `println!` line (and others) if it's just for debugging.

Reply:
fixed in 6628a685ce9995c2faddbae88c1e1baf322491b2

Reply:
fixed in afcd26032dfa039d9332b89bcd8b5db468f756a8